client/ayon_blender/api/pipeline.py

```diff
@@ -296,6 +296,15 @@ def set_resolution(data):
     scene.render.resolution_y = resolution_y
 
 
+def set_unit_scale_for_setting(unit_scale_settings=None):
```

Comment:
It's not setting it "for" a "setting" but from the settings?
```suggestion
def set_unit_scale_from_settings(unit_scale_settings=None):
```